other/refresh-env-var-in-pod/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -25,21 +25,23 @@ spec:
         file: deployment.yaml
   - name: step-03
     try:
+    - sleep:
```

Comment:
do we need to sleep here ?

Reply:
Strictly speaking? Maybe not. But it may help. And it's short.